infrastructure/networked_rds/README.md

```diff
@@ -0,0 +1,148 @@
+
+  project_name = "foodoasis"
+  stage = "dev"
+  region = "us-east-2"
+  account_id = "12345678910"
+
+  ssh_public_key_names = ["alice", "bob", "carol"]
+  availability_zones = ["us-east-2a", "us-east-2b", "us-east-2c"]
+
+  db_username = "foodoasis"
+  db_name = "foodoasis"
+  db_password = var.db_password
+}
+```
+
+## Inputs
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| project\_name | The overall name of the project using this infrastructure; used to group related resources by | `string` | `""` | yes |
+| account\_id | the aws account id # that this is provisioned into | `string` | `""` | yes |
+| stage | a short name describing the lifecyle or stage of development that this is running for; ex: 'dev', 'qa', 'prod', 'test'| `string` | `"dev"` | no |
+| region | the aws region code where this is deployed; ex: 'us-west-2', 'us-east-1', 'us-east-2'| `string` | `""` | yes |
+| cidr\_block |The range of IP addresses this vpc will reside in| `string` | `"10.10.0.0/16"` | no |
+| availability_zones |The region + identifiers for the zones that the vpc will cover. Ex: ['us-west-2a', 'us-west-2b', 'us-west-2c']. Varies between regions.| `string` | `""` | yes |
+| tags| key value map of tags applied to infrastructure | `map` | `{terraform_managed = "true"}` | no |
+| db\_username | The name of the default postgres user created by RDS when the instance is booted| `string` | `""` | yes |
+| db\_name |The name of the default postgres database created by RDS when the instance is booted | `string` | `""` | yes |
+| db\_password |The postgres database password created for the default database when the instance is booted. :warning: do not put this into git!| `string` | `""` | yes |
+| ssh\_public\_key\_names |the name of the public key files in ./public_keys without the file extension; example ['alice', 'bob', 'carol']| `list(string)` | `""` | yes |
+
+## Bastion server
+A [bastion
+server](https://docs.aws.amazon.com/quickstart/latest/linux-bastion/overview.html)
+is a hardened server through which access to resources running in private
+subnets of a VPC. An example use case is a database. Rather than create a
+database with ports open to the whole wide Internet we can create it within our
+own virtual cloud, and grant access to it via the bastion, aka "jumpbox", server.
+
+The bastion server here is composed from the very good [terraform community
+module](https://github.com/terraform-community-modules/tf_aws_bastion_s3_keys/tree/v2.0.0).
+
+To grant users access via the bastion to VPC resources simply add their SSH
+public key to the S3 bucket created by this blueprint. The server syncs its keys
+with that bucket hourly so that every time a key is added or removed, that users
+access is added or removed with it. 
+
+### bastion usage
+create a folder called `public_keys` in the directory from where you are
```

Comment:
please re-word or extend the examples here to a way that would have helped you better understand, as discussed on slack